### DISCUSSION TITLE
Deprecation of Docker Platform Tags

### DISCUSSION THREAD

**Discussion by lissy93** [Lissy93/dashy#481] [Breaking Change]
_Prior to version 2, if you were running Dashy on an ARM device, you had to pull an ARM-specific Docker image, by specifying the platform as a Docker tag (e.g. `:arm64v8`, `:arm32v7`)._ This is changing...

**There is now a single, multi-architecture image, that can be run on (almost) all system architectures.**

What does this mean for you?
If you're running Dashy on an ARM device, you must remove the platform tag from your compose file / run command, in order to keep the container up-to-date. You can either leave the tag off, or pin it to a specific version (e.g. `lissy93/dashy:2.0.0` or `:latest`)

How?
Under-the-hood, there have been a lot of changes to facilitate this. There's a brand-new Alpine-based [Dockerfile](https://github.com/Lissy93/dashy/blob/master/Dockerfile), and the building, testing and publishing process is now done with GitHub actions (using [this workflow](https://github.com/Lissy93/dashy/blob/master/.github/workflows/docker-build-publish.yml)).

Why?
These changes mean there are less images to maintain, and will reduce the financial cost of making Dashy available on DockerHub. The new image is also smaller.

When?
Version 2.0.0 / Jan 2022.

Any questions, feel free to ask below :)

**Comment by henkiewie** [comment]
Hi Lissy,
So I noticed just now. I already used the :latest for some time. Yesterday i was on 2.0.1. but now with (2.0.2) the infamous exec error displays in the log for the armv32v7 platform when using the :latest tag (I use raspberry PI Bullseye).
Luckily arm32v7 (:1.9.9 to :2.0.2) still work. So it must be there's no image image tag for arm32v7 pointing to the :latest tag. 
regards,
Henk
![Screenshot 2022-02-07 at 18 59 17](https://user-images.githubusercontent.com/74558874/152845364-567025e3-617c-4999-a94b-7ae9095efc4a.png)

**Reply by lissy93** [reply]
Hmm, oh no, that's not good. Let me look into this asap

**Reply by lissy93** [reply]
Okay, you are correct, found the issue...

For the image to be multi-arch, it needs to be build with qemu + buildx, which is what the GH workflow does, DockerHub doesn't support this. The issue was that I accidently left the old build hook in DockerHub, causing it to trigger a non-multi-arch build overwritting the latest tag 🤦‍♀️ 

It's fixed now, and I just tested `:latest` tag on my Pi w DietPi and it's working perfectly :)

Thanks so much for mentioning this, apologies for the trouble, big mistake on my part

**Reply by henkiewie** [reply]
Hi lIssy,
It's looking good now. Don't apologise. You acted so quick 🚀 , maybe nobody noticed 🤐 . Glad i could help. 😃
